### DISCUSSION TITLE
Collaboration Proposal with app.hyve.works

### DISCUSSION THREAD

**Discussion by vniregrbldskqncib3j43cjn4h54** [ringdao-community-guild/discussion#22] [General]
I was wondering if you guys are looking to expand the team because I wanted to introduce you to HYVE - a web3 freelancing platform.

The payments are instant and safe and always escrowed through smart contracts. HYVE is the first decentralized labor exchange, or in simpler terms, Uniswap for work and freelancing.
 
As we are now adding multiple payment methods and integrating more blockchains and tokens to have a vast portfolio for companies and people who want to pay freelancers in a specific token, I would propose integrating $RING as a payment method. Would that interest the team?

We are live on ETH, BSC, MATIC, and FTM, and we will launch the AVAX blockchain on our platform and you can list the token on one of these, plus on one of the future blockchain integrations. We partnered with over 150 crypto companies and have integrated over 100 tokens.
 
We would like to discuss more possibilities, so let us know what you think and we can discuss further on a call to ease the details exchange between us.

**Comment by Catinpijama** [comment]
Interesting proposal

**Reply by Ann66na** [reply]
yes, it is good

**Comment by JaffaSweet** [comment]
Very interesting

**Comment by Vilma487** [comment]
Interesting!

**Comment by ag2silver** [comment]
Have never heard about hyve

**Comment by wertum33** [comment]
Interesting, I want to know the details

**Comment by Fenjza** [comment]
Nice informaion

**Comment by kOLIANIKO** [comment]
Very good proposal!

**Comment by Weapon0x** [comment]
thx so much for info!

**Comment by kOLIANIKO** [comment]
This is very good proposal but I never heard about Hyve! need to check

**Comment by furoxr** [comment]
It's nice to see your proposal! I don't know much about HYVE, is it a open source project? Could you provide with more information or links to introduce HYVE? As for Darwinia Network, it's a public blockchain platform where developers can build decentralized applications and use the $RING token as a form of payment without requiring approval from anyone. Here are some resources about Darwinia Network:
1. [Documents](https://docs.darwinia.network/)
2. [RING info](https://www.coingecko.com/en/coins/darwinia-network)
3. [Explorer(subscan)](https://darwinia.subscan.io/)
4. [Explorer(polkadot.js)](https://cloudflare-ipfs.com/ipns/dotapps.io/#/explorer)

**Reply by vniregrbldskqncib3j43cjn4h54** [reply]
Thanks for it. Yeah, sure. I can provide you with more information.

[HYVE](https://app.hyve.works/) is the largest web3 job marketplace with payments that are escrowed through smart contracts, so money is instantly and securely transferred from one end to another. On HYVE you have the lowest fees perceived, so the structure is as it follows:
* 0% for jobs listed in $HYVE
* 2.5% for jobs listed in any cryptocurrency
There are 3 options available when posting on the platform and you can choose to freelance meaning that you will post your offer, or you can choose to hire someone for a specific task, or for a periodic engagement in the form of a job.

**Here you have some links:**
[Website](https://hyve.works/) 
[Platform](https://app.hyve.works/) 
[Twitter](https://twitter.com/join_hyve)
[Telegram](https://t.me/hyveworks)
[Blog](https://latest.hyve.works/)

Since you mentioned that, is it possible though to stay in touch with someone that can give me feedback for the integration article, and stay in touch for other future updates?

**Comment by vniregrbldskqncib3j43cjn4h54** [comment]
Since you were so enthusiastic about the proposal, we made things happen and we announced the integration a few moments ago. Check this out https://twitter.com/join_hyve/status/1674834752296058902?s=46&t=XAcP_5kkLQKNN9R4mUQGmg

**Reply by Laki389** [reply]
> Since you were so enthusiastic about the proposal, we made things happen and we announced the integration a few moments ago. Check this out https://twitter.com/join_hyve/status/1674834752296058902?s=46&t=XAcP_5kkLQKNN9R4mUQGmg

I'm not sure that the similar comments from bots "Very interesting", "Nice informaion", "Very good proposal!", "Interesting, thank you!" mean "enthusiasm" for the proposal... and the fact that you have already independently and unilaterally announced Twitter integration is the biggest red flag.
Also, on the following website https://latest.hyve.works/new-token-ring/ you have placed another contract for Ring on Polygon which further confirms that you are scammers.
The actual Ring Smart Contract Address(Polygon): is 0x9C1C23E60B72Bc88a043bf64aFdb16A02540Ae8f

**Reply by vniregrbldskqncib3j43cjn4h54** [reply]
Tbh I didn't know that, but I did try to get in contact and reach out to someone from the team and here I was guided to place the proposal. Since you mentioned that, maybe you can help me with that, and as well, thanks for noticing the contact mistake. Will do the edits accordingly to have the right Polygon address displayed.

**Comment by Laki389** [comment]
@furoxr 
IMHO, these people are scammers:
1. They flooded the comments with fake support, "Very interesting", "Nice informaion", "Very good proposal!", "Interesting, thank you!"
2. One-sidedly, they have already announced Twitter integration themselves, which is a big red flag.
3. on the following website https://latest.hyve.works/new-token-ring/ they have placed another contract for Ring on Polygon which further confirms that they are scammers.
The actual Ring Smart Contract Address(Polygon): is 0x9C1C23E60B72Bc88a043bf64aFdb16A02540Ae8f

**Reply by furoxr** [reply]
Thanks for the audit. I will close this discussion later. I find they have corrected the contract address.

**Comment by furoxr** [comment]
I don't think it's a good idea to attract attention by spamming. meaning that you can use its features without obtaining special permissions from anyone, but it doesn't necessarily indicate an official collaboration between us and Darwinia Network. We just help to provide some public information about Darwinia Network. If you encounter any problems while using Darwinia Network, feel free to start a new conversation and I'll be happy to assist you further.

**Comment by BritvaBo** [comment]
awesome. thanks

**Comment by binanceotsutki** [comment]
very interesting
